contactsbackup/README.md

```diff
@@ -0,0 +1,15 @@
+# Local Contacts Backup
+
+A backup application that backs up local on-device contacts via the system's backup API.
+This explicitly excludes contacts that are synced via sync accounts
+such as [DAVx⁵](https://www.davx5.com/).
+
+## Permissions
+
+* `android.permission.READ_CONTACTS` to back up local contacts.
+* `android.permission.WRITE_CONTACTS` to restore local contacts to the device.
+
+## License
+
+This application is available as Free Software
+under the terms of the [GPLv3 (or later) License](https://opensource.org/licenses/GPL-3.0).
```

Comment:
This will just make things complicated, Apache 2 is ideal for code that's meant for use in AOSP, like Seedvault.

Reply:
I agree with @chirayudesai. I think it's better to have the more permissive license.